bigquery/tables.rb

```diff
@@ -35,10 +34,9 @@ def list_tables project_id:, dataset_id:
   # project_id = "Your Google Cloud project ID"
   # dataset_id = "ID of the dataset to create table in"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.